internal/sensors/cmd.go

```diff
@@ -22,6 +22,10 @@ func (sensor *CmdSensor) GetId() string {
 	return sensor.Config.ID
 }
 
+func (sensor *CmdSensor) GetLabel() string {
+	return "CMD Sensor " + sensor.Config.ID
```

Comment:
Why the prefixes?
The ID must be unique, which should be validated as well.
Main gripe is that now its inconsistent between fan types :thinking:

Reply:
Well the usual fans have a label from the system like "hwmon3/fan1" so I thought in this case it makes sense to combine the type with the ID, so it doesn't look *exactly* the same as the ID.
However at least currently GetLabel() is not used for CMD or File fans or sensors (only used by `fan2go detect`), I just had to put *something* there so I could use the generic interface instead of `NvidiaFan`